worksheet.rb

```diff
@@ -31,4 +31,136 @@
 # - the total amount of money each driver has made
 # - the average rating for each driver
 # - Which driver made the most money?
-# - Which driver has the highest average rating?
\ No newline at end of file
+# - Which driver has the highest average rating?
+
+ride_log = { DR0004: [["3rd Feb 2016",5,"RD0022",5],
```

Comment:
Consider storing each individual trip as a hash rather than an array. For example
```
{date: "3rd Feb 2016", cost: 5, rider: "RD0022", rating: 5}
```
This will increase readability and make it so you don't need to rely on knowing which column the data is stored in, but can use the keys to index the values you need.